### PR TITLE
Pure js grpc

### PR DESCRIPTION
**Pull request by negiDharmendra**
Added support to generate a Gauge JS runner with pure js GRPC implementation which can be installed offline.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "lint": "eslint index.js src/ skel/ test/ scripts/",
     "test": "npm run lint && mocha --recursive",
     "installPlugin": "node scripts/install.js",
-    "package": "node scripts/install.js --package"
+    "package": "node scripts/install.js --package",
+    "offLinePackage": "node scripts/install.js --off-line-package"
```
Comment (nehashri) [inline package.json]: offline should be one word

File: scripts/install.js [hunk 2]
```diff
@@ -30,10 +30,29 @@ var recreateDir = function (dirPath) {
...
+  fs.copyFileSync(path.resolve("./offline-package.json"), path.resolve("./package.json"));
+};
+var prepareFiles = function (buildOffLinePakcage) {
   var buildDir = localPath("build"),
     copyList = ["gauge-proto", "src", "skel", "index.js", "index.bat", "debug.bat", "js.json", "package.json", "package-lock.json", ".node-inspectorrc", "README.md"];
+  if( buildOffLinePakcage) {
+    prepareOffLinePackageJSON();
+    try {
+      console.log("Installing dependencies...");
+      child_process.execSync("npm install", { cwd: localPath() });
```
Comment (nehashri) [inline scripts/install.js]: We don't need to install the dev dependencies.

File: package.json (hunk 1 above)
Comment (negiDharmendra) [reply package.json]: Fixed

File: scripts/install.js (hunk 2 above)
Comment (negiDharmendra) [reply scripts/install.js]: Fixed

**Pull request merged**